masterlist.yaml

```diff
@@ -17835,7 +17835,7 @@ plugins:
   - name: 'Water for ENB - Patch - FWMF Unmarked Old Map 4K.esp'
     after: [ 'FWMF SE Unmarked Old Map 4K.esp' ]
   # Standalone Skyrim Maps #
-  - name: '(Blackreach|Dayspring Canyon|Deepwood Vale|Forgotten Vale|Markarth|Skuldafn|Skyrim|Solstheim|Soul Cairn|Sovngarde)( Paper)? Map( by (FreelanceCartography|Geevee))? for FWMF\.esp'
+  - name: '(Apocrypha|Blackreach|Dayspring Canyon|Deepwood Vale|Forgotten Vale|Markarth|Skuldafn|Skyrim|Solstheim|Soul Cairn|Sovngarde)( Paper)? Map( by (FreelanceCartography|Geevee))? for FWMF( by (Limon))?\.esp'
```

Comment:
The inner brackets incapsulating `Limon` seem unnecessary, since the plugin name `Skuldafn Paper Map for FWMF by Limon.esp` doesn't include those brackets. As such using the outer brackets with the "Once or none" quantifier `?` after them should be enough.

`.. for FWMF( by Limon)?\.esp`